app/services/migration/migrators/user.rb

```diff
@@ -58,12 +58,12 @@ def call
           active_alert: npq_application.active_alert || user.active_alert,
           trn_verified:,
           created_at: ecf_user.created_at,
-          updated_at: ecf_user.updated_at,
+          significantly_updated_at: ecf_user.updated_at,
```

Comment:
Maybe we still can keep migrating `updated_at` over or not?

Reply:
discussed in person: let's keep both